### PR TITLE
homepage completed

### PR DESCRIPTION
**Pull request by JatuFaique**
Please Review Ecomm landing page

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -57,10 +64,26 @@
 }
 
 .featured-category {
+  position: relative;
   height: 300px;
```
Comment (Sharmaryan) [inline css/home.css:68]: use rem to make uniformity since you have used it in your entire code

**Review by Sharmaryan — APPROVED** [review]
suggested a change and UI looks great

File: css/home.css (hunk 1 above)
Comment (shashikantmaurya0007) [inline css/home.css:68]: if values are bigger than 10px use relative values like em,rem

**Review by shashikantmaurya0007 — APPROVED** [review]
Those are the few changes you need to take care of. one suggestion dont overuse position lot if things can be done with flex use flex

**Pull request merged**